assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -31,7 +31,7 @@ import { TEXT_ELEMENT_DEFAULT_FONT } from '../../app/font/defaultFonts';
 import Layout from '../../app/layout';
 import { DATA_VERSION } from '../../migration';
 import { createPage } from '../../elements';
-import FixtureEvents from './events';
+import FixtureEvents from '../../../../../karma/events';
```

Comment:
Maybe also let's put the `events.js`, `init.js` into a subfolder inside the `/karma/`? Maybe `/karma/fixture` as well?

Reply:
Do you mean putting `event.js`, `init.js` into a subfolder named `karma/fixture` like so:

![file-dir](https://user-images.githubusercontent.com/2755722/85629139-4aa8e900-b63f-11ea-9c3d-b9a0ae0407d0.png)

Reply:
Yep. Exactly.